### PR TITLE
Let there be scroll bars

### PR DESCRIPTION
**Pull request by LongCatIsLooong**
## Description

Add `Scrollbar`s to Gallery. Most scrollables without sliver navigations now have a scroll bar. All  changes are trivial. The GalleryOptionsPage now has a hard coded bottom padding.

## Related Issues

#5525

## Tests

No tests yet.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] All existing and new tests are passing.
- [x] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Does your PR require Flutter developers to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (Please read [Handling breaking changes]). *Replace this with a link to the e-mail where you asked for input on this proposed change.*
- [x] No, this is *not* a breaking change.

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

**Comment by HansMuller** [comment]
I was a bit concerned about unconditionally adding scrollbars everywhere, but that seems to be OK.

File: examples/flutter_gallery/lib/gallery/options.dart [hunk 1]
```diff
@@ -441,31 +441,37 @@ class GalleryOptionsPage extends StatelessWidget {
...
-          <Widget>[
-            const Divider(),
-            const _Heading('Flutter gallery'),
-            _ActionItem('About Flutter Gallery', () {
-              showGalleryAboutDialog(context);
-            }),
-            _ActionItem('Send feedback', onSendFeedback),
-          ],
+      child: MediaQuery(
+        data: data.copyWith(padding: data.padding.copyWith(bottom: 94)),
```
Comment (xster) [inline examples/flutter_gallery/lib/gallery/options.dart]: I wouldn't add this one

**Review by xster — COMMENTED** [review]
LGTM

File: examples/flutter_gallery/lib/gallery/options.dart (hunk 1 above)
Comment (HansMuller) [reply examples/flutter_gallery/lib/gallery/options.dart]: I agree, this is only place where scrollbars probably don't belong.

**Review by HansMuller — APPROVED** [review]
LGTM with one less scrollbar

**Pull request merged**